src/visual.ts

```diff
@@ -332,6 +332,9 @@ module powerbi.extensibility.visual {
                         description: columns.EventDescription && columns.EventDescription.values && <string>columns.EventDescription.values[categoryIndex],
                     };
                 }
+                let y_value = <number>(y_group0Values && y_group0Values[categoryIndex]) || <number>(y_group1Values && y_group1Values[categoryIndex]) || 0;
```

Comment:
Do unit tests cover changes that you have made?

Reply:
Yes

Reply:
I don't see any unit-tests in this PR

Reply:
Increase version of the visual and update changelog

Reply:
Done